dev/io.openliberty.ws.jaxws.tools_fat/fat/src/com/ibm/ws/jaxws/tools/FATSuite.java

```diff
@@ -33,5 +33,5 @@
 public class FATSuite {
 
     @ClassRule
-    public static RepeatTests r = RepeatTests.withoutModificationInFullMode().andWith(FeatureReplacementAction.EE8_FEATURES().fullFATOnly()).andWith(FeatureReplacementAction.EE9_FEATURES());
+    public static RepeatTests r = RepeatTests.withoutModificationInFullMode().andWith(FeatureReplacementAction.EE8_FEATURES().fullFATOnly()).andWith(FeatureReplacementAction.EE9_FEATURES().fullFATOnly()).andWith(FeatureReplacementAction.EE10_FEATURES());
```

Comment:
This won't work when run with Java 8.  You will need to do the conditionalFullFatOnly method like you can see in the example fat project.

Reply:
```suggestion
    public static RepeatTests r = RepeatTests.withoutModificationInFullMode().andWith(FeatureReplacementAction.EE8_FEATURES().fullFATOnly()).andWith(FeatureReplacementAction.EE9_FEATURES().conditionalFullFATOnly(FeatureReplacementAction.GREATER_THAN_OR_EQUAL_JAVA_11)).andWith(FeatureReplacementAction.EE10_FEATURES());
```